parsers/GB.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+# coding=utf-8
+
+"""
+Parser that uses the RTE-FRANCE API to return the following data type(s)
+fetch_price method copied from FR parser.
+Day-ahead Price
+"""
+
+from datetime import timedelta
+import arrow
+import logging
+import os
+
+import requests
+import xml.etree.ElementTree as ET
+
+from parsers.lib.config import refetch_frequency
+
+@refetch_frequency(timedelta(days=1))
+def fetch_price(zone_key, session=None, target_datetime=None,
+                logger=logging.getLogger(__name__)) -> list:
+    if target_datetime:
+        now = arrow.get(target_datetime, tz='Europe/London')
+    else:
+        now = arrow.now(tz='Europe/London')
+
+    r = session or requests.session()
+    formatted_from = now.shift(days=-1).format('DD/MM/YYYY')
+    formatted_to = now.format('DD/MM/YYYY')
+
+    url = 'http://www.rte-france.com/getEco2MixXml.php?type=donneesMarche&da' \
+          'teDeb={}&dateFin={}&mode=NORM'.format(formatted_from, formatted_to)
+    response = r.get(url)
+    obj = ET.fromstring(response.content)
+    datas = {}
+
+    for donnesMarche in obj:
+        if donnesMarche.tag != 'donneesMarche':
+            continue
+
+        start_date = arrow.get(arrow.get(donnesMarche.attrib['date']).datetime, 'Europe/London')
```

Comment:
I have a doubt here if I should put 'Europe/London' or 'Europe/Paris' for the timezone.

Reply:
That is a good question 😄 
Let's see how the data looks now and then we can follow-up on it if it's clear that it's off :)